package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@microlink/react-json-view",
   "description": "Interactive react component for displaying javascript arrays and JSON objects.",
   "homepage": "https://github.com/microlinkhq/react-json-view",
-  "version": "1.22.1",
+  "version": "1.22.2",
```

Comment:
please don't increment the version. It will be done automatically after merge the PR